packtools/sps/models/funding_group.py

```diff
@@ -9,6 +9,13 @@ def _is_funding_source(text):
     return all(char.isalpha() or char.isspace() for char in text)
 
 
+def _strip_and_remove_final_dot(text):
```

Comment:
@Rossi-Luciano Os modelos tem pelo menos duas principais funcionalidades: a) facilitar a obtenção dos dados; b) ser fiéis para que a validação seja justa.
No caso a), é ok, realizar a remoção do ponto final, no entanto, para a finalidade b) estaria adulterando o dado.
Sendo assim, acho que seria conveniente ter duas saídas, dois métodos diferentes, um fiel e outro que ajusta o dado.

Reply:
OK, @robertatakenaka